civic.json

```diff
@@ -0,0 +1,8 @@
+{
+    "status": "Beta", 
```

Comment:
You're right, I'll update it. We are testing out our new [Project Search](http://www.codeforamerica.org/brigade/projects) page and so are adding in all this metadata about our projects. This round was more about seeing if our tool actually worked. Sorry I got some of the info incorrect.